ishell/__init__.py

```diff
@@ -3,5 +3,5 @@
 
 from .log import logger
 
-__version__ = "0.1.6"
+__version__ = "0.1.7.1"
```

Comment:
I just realized that 0.1.7.1 was already used by systems, so let's bump this up to 0.1.7.2